### PR TITLE
ICMCLST-1662 Add FA-DFL licence data serializer

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: mail/enums.py [hunk 1]
```diff
@@ -132,7 +134,9 @@ def OPEN_GENERAL_LICENCES(cls):
     LicenceTypeEnum.OGCL.value: "OGE",
     LicenceTypeEnum.OGTL.value: "OGE",
     # ICMS Licence types
+    LicenceTypeEnum.IMPORT_DFL.value: "SIL",
```
Comment (mikeylockwood) [inline mail/enums.py:137]: DFL

Comment (MattHolmes123) [reply mail/enums.py:137]: It's by design, CHIEF treats the two ICMS application types as the same.

File: mail/tests/test_builders.py [hunk 2]
```diff
@@ -136,3 +136,75 @@ def test_generate_icms_licence_file(self):
...
+                "line_3": "line_3",
+                "line_4": "line_4",
+                "line_5": "",
+                "postcode": "S881ZZ",
+            },
+        }
+        restrictions = (
+            "Not valid for items originating in or consigned from Iran, North"
+            " Korea, Libya, Syria or the Russian Federation.(including any"
+            " previous name by which these territories have been known)."
```
Comment (saruniitr) [inline mail/tests/test_builders.py:159]: unless it is public information may be replace with example text?

Comment (MattHolmes123) [reply mail/tests/test_builders.py:159]: I can replace it.

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**